### ISSUE TITLE
Remote branch -> Remote-tracking branch

### ISSUE THREAD

**Issue by YueLinHo** [progit/progit2#301]
For ```(remote)/(branch)```,

the command ```git branch -h``` shows
```
...
Generic options
...
    -r, --remotes         act on remote-tracking branches
...
Specific git-branch actions:
    -a, --all             list both remote-tracking and local branches
...
```

So, some ```Remote branch``` should be ```Remote-tracking branch``` in progit2/book/03-git-branching/sections/remote-branches.asc

**Comment by ben** [comment]
Want to become a contributor? :smile:  Open up [the file](https://github.com/progit/progit2/blob/master/book/03-git-branching/sections/remote-branches.asc) and click the "Edit" ( :pencil2: ) button at the top, and submit a pull request.

**Comment by YueLinHo** [comment]
@ben Thanks for your information. Perhaps I will give it a try later. :wink:

**Comment by YueLinHo** [comment]
Fixed in 3378e04d7d1c3f95eec3642ec54caf2821050948
